Microsoft/WebTemplateStudio
# Removed orange highlight from selections onClick but kept for tab

To Test:
- Confirm when not currently tabbed the selections (Frameworks + Pages) keep a white outline.
- Confirm when tabbed they become orange.
- Confirm that all cards (SelectableCards) haven't lost any intended highlight, whether it be white or 
orange!

Changes:
- Added polyfill for focus-visible, and used its feature to have selective orange highlight.

@xtinah-w is this what you wanted with #1086 ?  (Build my branch, or just run the client after going into my branch).
closes #1086
